web-marketplace/src/components/organisms/DetailsSection/DetailsSection.tsx

```diff
@@ -56,7 +61,19 @@ export const DetailsSection: React.FC<
                 {header.title}
               </Title>
               <Body size="lg" mobileSize="md" maxWidth={718}>
-                <BlockContent content={header.descriptionRaw} />
+                {IS_REGEN ? (
+                  <BlockContent content={header.descriptionRaw} />
+                ) : (
+                  <div>
+                    {_(TERRASOS_PROJECT_DESCRIPTION)}
+                    <Link
```

Comment:
nothing happens when I click on the link
this seems to be related to another component overlapping

Reply:
Indeed, it's fixed now.